.dockerignore

```diff
@@ -0,0 +1,12 @@
+fly.toml
+.git/
+
+# DB
+*.sqlite3
+
+# Virtual Envs
+.venv/
+venv/
+
+# Static Files
+static/
```

Comment:
I think we should add `theme/static/` here, too. Tailwind places its output there and we should probs keep that out of the image.